frontend/src/app/authentication/authenticated.component.ts

```diff
@@ -64,6 +66,19 @@ export class AuthenticatedComponent implements OnInit {
   setShowSUDS(user) {
     this.showSUDS = user && user.role === 'admin' && (!user.forests || user.forests.length === 0);
   }
+
+  /**
+   * determine user role, and display applications if correct role
+   * set route for viewing applications
+   */
+  adminOrUser() {
```

Comment:
This should be separated into 2 separate methods that each return the correct information. As it stands:
1. its brittle as it requires that `adminOrUser` be called before `useRoute` is referenced
2. The expectation is that is returns a boolean, but actually returns a "truthy" value or `false` thats not totally clear.
3. There are 2 different pieces of logic. First, whether to show the link, which only depends on `specialUse` (and maaaaybe if there actually is a user), and second, what href to use which depends on the user role.
4. The logic in `ngOnInit` implies that a user may not be present, which will cause an error when this renders so we should make sure to handle that case.